modules/tofi/hm.nix

```
@@ -0,0 +1,34 @@
+{
+  config,
+  lib,
+  ...
+}:
+with config.stylix.fonts;
+with config.lib.stylix.colors.withHashtag; {
+  options.stylix.targets.tofi.enable =
+    config.lib.stylix.mkEnableTarget "Tofi" true;
+
+  config = lib.mkIf config.stylix.targets.tofi.enable {
+    programs.tofi.settings = let
+      background = base00;
+      foreground = base05;
+      darkForeground = base04;
+      selection = base03;
+    in {
+      font = monospace.name;
+      font-size = "${toString sizes.popups}";
```

This string interpolation is redundant: 

```suggestion
      font-size = toString sizes.popups;
```